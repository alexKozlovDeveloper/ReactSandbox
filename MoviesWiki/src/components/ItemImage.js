import React, { Component } from "react";

class ItemImage extends Component {
    render() {
        return (
            <div>
                <div>Image {this.props.image}</div>
            </div>
        );
    }
}

export default ItemImage;