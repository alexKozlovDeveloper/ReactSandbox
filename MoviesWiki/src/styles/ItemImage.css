.image {
    width: 100%;
    background-color: #321b56;
}

.imagecontainer {

}