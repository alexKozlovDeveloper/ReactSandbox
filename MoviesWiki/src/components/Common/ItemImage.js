import React, { Component } from "react";

import styles from "../../styles/ItemImage.css"

function ItemImage(props) {
    return  <div className={styles.imagecontainer}>
                <img className={styles.image} src={props.image}></img>
            </div>;
}

export default ItemImage;