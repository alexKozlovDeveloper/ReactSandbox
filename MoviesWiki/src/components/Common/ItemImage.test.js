import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import ItemImage from "./ItemImage";

configure({adapter: new Adapter()});

test('ItemImage rendering test', () => {

    var image = "someImage";

    const component = shallow(<ItemImage image={image} />);
    expect(shallowToJson(component)).toMatchSnapshot();
 });

