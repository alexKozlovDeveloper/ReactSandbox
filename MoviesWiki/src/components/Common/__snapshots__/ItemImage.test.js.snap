// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemImage rendering test 1`] = `
<div
  className="imagecontainer"
>
  <img
    className="image"
    src="someImage"
  />
</div>
`;
